### ISSUE TITLE
Use choco fix notation with Force parameter

### ISSUE THREAD

**Issue by majkinetor** [majkinetor/au#8]
@ferventcoder
>  If I update with force and the package exists at the end point, will it automatically use package fix version notation? Or can I provide an option to do that?

**Comment by majkinetor** [comment]
iteration only when there is revision number otherwise use date as revision

**Comment by majkinetor** [comment]
Impelmented variant of it:  

- choco fix version always go in the revsion 
- existing fixed versions are changed to contain current date if the revision does not exist or it already contains choco fix
- if revision is there and not in choco fix format, just keep existing one but write info about i.

Example with existing 'choco fix revision'

```
PS C:\Work\chocolatey\copyq> .\update.ps1
copyq - checking updates
nuspec version: 2.7.1.20160708
remote version: 2.7.1
No new version found, but update is forced
Automatic checksum started
Downloading copyq 32 bit
  from 'https://github.com/hluk/CopyQ/releases/download/v2.7.1/copyq-2.7.1-setup.exe'

Download of copyq-2.7.1-setup.exe (9.91 MB) completed.
Package downloaded and hash calculated for 32 bit version
Updating files
  copyq.nuspec
    updating version using Chocolatey fix notation: 2.7.1.20160708 -> 2.7.1.20160814
```

Example without revision:
```
cpu-z.install - checking updates
nuspec version: 1.77
remote version: 1.77
No new version found, but update is forced
Automatic checksum started
Downloading cpu-z.install 64 bit
  from 'http://download.cpuid.com/cpu-z/cpu-z_1.77-en.exe'

Download of cpu-z_1.77-en.exe (1.63 MB) completed.
Package downloaded and hash calculated for 32 bit version
Updating files
  cpu-z.install.nuspec
    updating version using Chocolatey fix notation: 1.77 -> 1.77.0.20160814
```

Example with non choco fix revision:

```
PS C:\Work\chocolatey\dngrep> .\update.ps1
dngrep - checking updates
nuspec version: 2.8.16.0
remote version: 2.8.16.0
No new version found, but update is forced
Updating files
  dngrep.nuspec
    version not changed as it already uses 'revision': 2.8.16.0
```

The problem with 00, 01 etc.. fix is that each subsequent run will increment it as I don't know if its set today or any other day. This will cover most of the cases any way, others can be handled manually.

**Comment by ferventcoder** [comment]
LGTM
